plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4413,6 +4448,25 @@ else if (mechanism.requireObject(EntityTag.class)) {
                 spawner.update();
             }
         });
+
+        // <--[mechanism]
+        // @object LocationTag
+        // @name last_interacted_slot
+        // @input ElementTag(Number)
+        // @description
+        // Sets the last interacted slot of a Chiseled Bookshelf inventory.
+        // @tags
+        // <LocationTag.last_interacted_slot
+        // -->
+        tagProcessor.registerMechanism("last_interacted_slot", false, (object, mechanism) -> {
```

Comment:
Should use the `registerMechanism` method that takes a param type (and use that param in the code instead of `Mechanism#getValue`.